amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/bulk/hold/HoldBulkMonitor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * #%L
+ * Alfresco Records Management Module
+ * %%
+ * Copyright (C) 2005 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * -
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ * -
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.module.org_alfresco_module_rm.bulk.hold;
+
+import java.util.List;
+
+import org.alfresco.module.org_alfresco_module_rm.bulk.BulkMonitor;
+import org.alfresco.rm.rest.api.model.HoldBulkStatus;
+
+/**
+ * An interface for monitoring the progress of a bulk hold operation
+ */
+public interface HoldBulkMonitor extends BulkMonitor<HoldBulkStatus>
+{
+    /**
+     * Get the batch statuses for a hold
+     *
+     * @param holdId the hold id
+     * @return the batch statuses
+     */
+    List<HoldBulkStatus> getBatchStatusesForHold(String holdId);
```

Comment:
We always used the term `bulk` until now, is there a particular reason why we're referring to a `batch` here instead? If not, I'd suggest adhering to the semantic convention we just made up to avoid confusion.

Reply:
This is leftover. Changed. Now we are referring to a `batch`.